roles/bootstrap-os/tasks/setup-pipelining.yml

```diff
@@ -0,0 +1,6 @@
+---
+# Remove requiretty to make ssh pipelining work
+
+- name: Remove require tty
+  lineinfile: regexp="^\w+\s+requiretty" dest=/etc/sudoers state=absent
```

Comment:
You can't run this command before bootstrapping ubuntu or coreos on a remote host. It will fail without a python interpreter.